src/index.ts

```diff
@@ -51,15 +49,11 @@ export function createScheduler(): Scheduler {
   }
 
   function flush() {
-    do {
-      for (const task of queue) {
-        task();
-        served.add(task);
-        queue.delete(task);
-      }
-    } while (queue.size > 0);
+    for (const task of queue) {
+      task();
+    }
 
-    served.clear();
+    queue = new Set();
```

Comment:
I'd love to know if this is true because `.clear()` is extremely efficient as far as I know.

Reply:
Interesting
I chacked FF, Safari and Chrome - only in chrome creating new instance is faster (~5x)

In Safary .clear() faster (up to 3x) and in FF clear much much faster.

So, I'll keep clear()

Reply:
@mihar-22 I used .clear() now

Reply:
Thanks heaps for testing that @oleggrishechkin!